app/uk/gov/hmrc/nationalinsurancerecord/connectors/NpsConnector.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nationalinsurancerecord.connectors
+
+import uk.gov.hmrc.nationalinsurancerecord.domain.{NationalInsuranceRecord, NationalInsuranceTaxYear}
+
+import scala.concurrent.Future
+
+trait NpsConnector {
+
+  def getNationalInsuranceRecord: Future[NationalInsuranceRecord]
+  def getTaxYear: Future[NationalInsuranceTaxYear]
```

Comment:
Should this method exist? NPS does not have a single tax year endppoint.
NationalInsuranceService should pull a NationalInsuranceTaxYear out from the NpsNIRecord that comes back.